docs/docs/quickstart.md

```diff
@@ -152,8 +180,6 @@ TWITTER_COOKIES='[{"key":"auth_token","value":"your token","domain":".twitter.co
   {"key":"guest_id","value":"your guest_id","domain":".twitter.com"}]'
 ```
 
-Using TWITTER_COOKIES makes providing TWITTER_PASSWORD and TWITTER_EMAIL unnecessary. TWITTER_USERNAME is still required.
```

Comment:
was removing this intentional and if so why?

Reply:
twitter cookies won't work without email and password until this pr https://github.com/ai16z/eliza/pull/856 is merged.

Reply:
Btw, I think you might checkout your branch based on an old commit from main branch. Since this sentence was already removed in https://github.com/ai16z/eliza/pull/872/files

Reply:
I thought I was merging with the latest code but it might have been off of main.